packages/client/src/api.ts

```diff
@@ -54,6 +96,13 @@ export abstract class MessagingChannelApi extends MessagingChannelBase {
     }, undefined)
   }
 
+  /**
+   * Lists the conversations that a user participates in
+   * @param clientId id of the client that owns the user
+   * @param userId id of the user that owns the conversations
```

Comment:
It owns the conversations or participate in?

Reply:
good catch